xknx/remote_value/remote_value.py

```diff
@@ -58,7 +63,11 @@ def __del__(self):
     @property
     def initialized(self):
         """Evaluate if remote value is initialized with group address."""
-        return bool(self.group_address_state or self.group_address)
+        return bool(
+            self.group_address_state
+            or self.group_address
+            or len(self.listening_group_addresses)
```

Comment:
```suggestion
            or self.listening_group_addresses
```
an empty list is falsey